source/sgp_mode/SGPDataNodes.h

```diff
@@ -12,7 +14,13 @@ void SGPWorld::CreateDataFiles() {
       .SetTimingRepeat(my_config->DATA_INT());
   SetupTasksFile(my_config->FILE_PATH() + "Tasks" + my_config->FILE_NAME() +
                  file_ending)
-      .SetTimingRepeat(my_config->DATA_INT());
+        .SetTimingRepeat(my_config->DATA_INT());
```

Comment:
Which task file is set up should be done based on the configuration setting so that we don't have empty files for the other task